docs/apidiffs/current_vs_latest/opentelemetry-exporter-jaeger.txt

```diff
@@ -1,2 +1,1372 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
+---! REMOVED CLASS: PUBLIC(-) FINAL(-) io.opentelemetry.exporter.jaeger.proto.api_v2.Collector  (not serializable)
```

Comment:
@jkwatson We accidentally had these in the public API. We need to make a semconv decision on keeping these or not. I attempted to keep without any change to the API but had trouble doing it without also vendoring in `google.protobuf.Timestamp` / `google.protobuf.Duration`. I can try harder but wondering if this could be considered an OK breaking change semconv wise since it seems unlikely they're used (and we could patch them back in if it caused a problem).

Reply:
I think we definitely don't want to consider some proto classes as a part of our public APIs to support. I'm ok with hiding this class, and, as you said, if we have complaints we can add them in a patch release. I'd rather do that than expose all of those protobuf classes.